build.sh

```diff
@@ -208,10 +208,27 @@ do
         GROUP_ID=50
       fi
       docker build -t avro-build-${USER_NAME} - <<UserSpecificDocker
-FROM avro-build
+FROM docker.io/surajacharya/avro-build-test
```

Comment:
Yes. Once there is a general consensus that the base image being used is good enough I intend to move it to a more central location (apache docker hub) and point it to that.